kb-api-ws/src/main/java/dk/kb/api/webservice/ApiServiceImpl.java

```diff
@@ -24,6 +24,7 @@ public ApiServiceImpl(){
         rest = new RestUtil(solrUrl);
     }
 
+    //Acts as a proxy to a backing Solr and returns the result directly
```

Comment:
Description of methods should be JavaDoc-style `/** ... */` (you can leave out the parameter-description if they are self-explanatory), obviously for the sake of generating JavaDocs, but also because IDEs such as IDEA uses them while coding (try positioning the cursor over a method-call and press CTRL-q).

Reply:
[https://www.jetbrains.com/help/idea/working-with-code-documentation.html#](url) Gives an easy shortcut to creating this